File: find-the-original-typed-string-i.ts
{
    function possibleStringCount(word: string): number {
        let c = 1;

        for (let i = 1; i < word.length; i++) {
            if (word[i - 1] == word[i])
                c++;
        };

        return c;
    };

    console.log(possibleStringCount('abbcccc'))
}

File: find-the-original-typed-string-i.rs
pub fn possible_string_count(word: String) -> i32 {
    let mut c = 1;
    let chars: Vec<char> = word.chars().collect();

    for i in 1..chars.len() {
        if chars[i - 1] == chars[i] {
            c += 1;
        }
    }

    c
}
